web/client/examples/rasterstyler/assets/css/viewer.css

```diff
@@ -0,0 +1,108 @@
+html, body, #container, .fill {
```

Comment:
I think all these css are not needed, just include one css with the needed stuff